tests/Fixer/Alias/EchoToPrintFixerTest.php

```diff
@@ -47,8 +50,9 @@ public function provideFixCases()
                 print("test");
                 ',
             ),
-            // echo can take multiple parameters (although such usage is rare) while print can take one argument
-            // http://www.w3schools.com/php/php_echo_print.asp
```

Comment:
thanks for that :)
I just felt the official documentation is more complete and reliable, but yes this can be reverted.